.circleci/config.yml

```diff
@@ -188,30 +188,30 @@ jobs:
       - run:
           name: Ensure existence of Groth parameters and keys on remote host
           command: |
-            ./fil-proofs-tooling/scripts/paramcache-remote.sh "${CIRCLE_BRANCH}" "${BENCHMARK_SERVER_SSH_USERNAME}@${BENCHMARK_SERVER_IP_ADDR}" "-z=$((1024*1024*1024))"
+            ./fil-proofs-tooling/scripts/run-remote.sh "${CIRCLE_BRANCH}" "${BENCHMARK_SERVER_SSH_USERNAME}@${BENCHMARK_SERVER_IP_ADDR}" cargo run --release --package filecoin-proofs --bin=paramcache -- -z=$((1024*1024*1024))
```

Comment:
I can see not changing this with the PR, but in general I think it's a good idea to use long parameter names in scripts. things like `-z` are just so typo-prone.

Reply:
Just updated to use the long parameter name, as per your suggestion @vmx